crml/governance/src/lib.rs

```diff
@@ -97,12 +124,18 @@ decl_storage! {
 		ProposalVotes get(fn proposal_votes): map hasher(twox_64_concat) ProposalId => ProposalVoteInfo;
 		/// Map from proposal Id to status
 		ProposalStatus get(fn proposal_status): map hasher(twox_64_concat) ProposalId => Option<ProposalStatusInfo>;
+		/// Map from proposal Id to referendum votes
+		ReferendumVoteCount get(fn referendum_votes): double_map hasher(twox_64_concat) ProposalId, hasher(twox_64_concat) T::AccountId => ReferendumVotes;
```

Comment:
can use this to track how accounts voted
instead of counting votes at the end it's easier to keep a `map: address => u32`  and increment on every no vote